posts/2021-04-19-JAXRS-multipart-payloads-21005-beta.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta - OpenLiberty.io
+seo-description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for your JAX-RS client and services and also present in this release is Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  
+blog_description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for your JAX-RS client and services and also present in this release is Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.   
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+With Open Liberty 21.0.0.5-beta your can now build multipart payloads for your JAX-RS client and services and also present in this release is Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
```

Comment:
```suggestion
* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
```

Suggestion to reverse the order of these bullets - mainly so that it clarifies what the "All Beta" is larger than.

Reply:
@mbroz2 wdyt?

Reply:
I actually want to keep it as is, with allbeta first as I expect that one to get more downloads and because we start with the non-jakartaee9 updates in the post so it better mirrors the content.
We could drop 'larger' from it's description while keeping 'lightweight' in the jakarta one (or change to 'smaller'), but I'm not sure that's necessary either.

Reply:
Will leave as is for now.